.github/ISSUE_TEMPLATE/02-testgap.yml

```diff
@@ -0,0 +1,49 @@
+name: Test Gap Issue Template
+description: File a test gap issue.
+title: "[Test Gap][<Feature:>][<Topo:>]<Test Gap Title>"
+labels: ["Test Gap"]
+projects: ["sonic-mgmt"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this test gap issue!
+
+        If you are reporting a new test gap, make sure that we do not have any duplicates already open. You can ensure this by searching the issue list in this repository. If there is a duplicate, please close your issue and add a comment to the existing issue instead.
+
+  - type: markdown
+    id: tc-contact-links
+    attributes:
+      value: |
+        Please ask and answer questions here at the [SONiC Support Forums](https://groups.google.com/forum/#!forum/sonicproject)
+
+  - type: textarea
+    id: tc-description
+    attributes:
+      label: Test Gap Description
+      description: Describe the test gap in the test coverage.
+      placeholder:
+        Why do we need this test gap? What part of the functionality is not covered by existing tests?
+        Explain the impact of this test gap. What potential issues or risks could introduce from not having this test?
+    validations:
+      required: true
+
+  - type: markdown
+    id: tc-test-environment
+    attributes:
+      label: Test environment
+      description: Provide details about the test environment where the test gap was identified.
+      placeholder:
+        * SONiC version
```

Comment:
Is it better to leave a blank line for use to fill each of thoese fields?

Reply:
placeholder is just a hint for expected input, it won't persist after actually filling in text.